discovery-provider/src/queries/get_tracks.py

```diff
@@ -136,12 +145,12 @@ def get_tracks(args: GetTrackArgs):
         def get_tracks_and_ids():
             if "handle" in args:
                 handle = args.get("handle")
-                user_id = (
+                user = (
                     session.query(User.user_id)
                     .filter(User.handle_lc == handle.lower())
                     .first()
                 )
-                args["user_id"] = user_id
+                args["user_id"] = user.user_id
```

Comment:
for future reader -- this change was necessary for the above comparison. this previously was pulling a (user_id, ) tuple, but was working b/c sql alchemy is smart. this is more correct